style.css

```diff
@@ -0,0 +1,507 @@
+  top: 530px;
+  left: 120px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img6 {
+  position: absolute;
+  top: 530px;
+  left: 305px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation-img7 {
+  position: absolute;
+  top: 530px;
+  left: 500px;
+  line-height: 1.5;
+  width: 228px;
+  word-spacing: -1.5px;
+  font-size: 11px;
+  margin-top: -13px;
+  text-align: left;
+}
+
+.inflation h2 {
+  margin-top: 20px;
+  text-align: left;
+  font-size: 15px;
+}
+
+.inflation-p {
+  font-size: 15px;
+  color: grey;
+  line-height: 18px;
+  margin-top: -10px;
+  text-align: left;
+}
+
+.body p {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+button {
+  height: 30px;
+  align-content: right;
+  border-radius: 5px;
+  background-color: #6287a5;
+  color: white;
+  font-weight: 100;
+}
+
+.button {
+  float: left;
+  margin-right: 350px;
+  margin-top: -10px;
+}
+
+.button1 {
+  width: 150px;
+  margin-left: 350px;
+  margin-top: -50px;
+}
+
+.button2 {
+  width: 90px;
+}
+
+.date {
+  float: left;
+  margin-right: 350px;
+  font-size: 12px;
+}
+
+#footer ul {
+  float: left;
+  margin-left: 80px;
+  margin-bottom: 30px;
+}
+
+#dennis {
+  font-weight: bold;
+  margin-bottom: -20px;
+  font-size: 12px;
+}
+
+.inflation {
+  text-align: center;
+}
+
+div .inflation {
+  font-size: 14px;
+  width: 1000px;
+}
+
+.quick-links > p,
+.quick-links > hr,
+.aside > h3 {
+  margin-left: 50px;
+}
+
+.footer-links hr {
+  margin-top: 20px;
+}
+
+div .inflation hr {
+  width: 50px;
+  margin-top: -10px;
+  margin-left: -5px;
+}
+
+.div3 {
+  align-content: center;
+  font-size: 1.25rem;
+  line-height: 1.875rem;
+  padding: 350px;
+  padding-top: 20px;
+  padding-bottom: 30px;
+}
+
+.div3 img {
+  width: 650px;
+}
+
+@font-face {
+  font-family: 'Title';
+  src: url('./fonts/Durwent.ttf') format('embedded-opentype');
+}
+
+.one {
+  text-align: center;
+  margin-top: 5px;
+}
+
+.one h1 {
+  margin-top: 100px;
+  text-align: center;
+}
+
+.one img {
+  height: 500px;
+  width: 800px;
+}
+
+.one p {
+  float: left;
+  text-align: left;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  margin-right: 7px;
+  margin-left: 280px;
+  color: #666;
+  font-family: 'nyt-imperial', georgia, 'times new roman', times, serif;
+}
+
+ul li {
+  list-style: none;
+}
+
+footer {
+  margin-top: -30px;
+  font-family: 'nyt-franklin', Arial, Helvetica, sans-serif;
+  clear: both;
+}
+
+footer ul li {
+  line-height: 22px;
+  font-size: 12px;
+}
+
+.footer-links ul li {
+  display: inline-block;
+  margin-left: 25px;
+}
+
+.social ul li {
+  display: inline;
+  color: #ccc;
+  margin-right: 10px;
+}
+
+#social ul li {
+  display: inline;
+  margin-right: 10px;
+  color: #ccc;
+}
+
+.Rick {
+  font-size: 11px;
+  margin-top: -10px;
+  color: grey;
+}
+
+.div1 #search {
+  flex: left;
```

Comment:
[HINT]
Flex is not a value for flex. Check [this](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) link.